src/components/exchange/ExchangeField.tsx

```diff
@@ -67,18 +99,22 @@ const ExchangeField = (
   },
   ref
 ) => {
+  const inputRef = ref as MutableRefObject<TextInput>;
   const { colors } = useTheme();
   const [value, setValue] = useState(amount);
 
-  const colorForAsset = useColorForAsset({
-    address,
-    fallbackColor: colors.appleBlue,
-    mainnet_address: mainnetAddress,
-    type: mainnetAddress ? AssetType.token : type,
-  });
+  const colorForAsset = useColorForAsset(
+    {
+      address,
+
+      mainnet_address: mainnetAddress,
+      type: mainnetAddress ? AssetType.token : type,
+    },
+    colors.appleBlue
+  );
```

Comment:
we're missing something here, ETH color is appleBlue by default https://www.loom.com/share/0bb656e7673144de9740ab1021aed9ae

Reply:
p shit api but fallbackColor should be `address ? undefined : colors.appleBlue` same as `TokenSelectionButton`

Reply:
could also prob just remove for this case

Reply:
Using `address ? undefined : colors.appleBlue` so at least we're consistent